src/DAOApi.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import type { Headers } from 'brewskey.js-api';
+
+import oHandler from 'odata';
+
+import AccountDAO from './dao/AccountDAO';
+import AvailabilityDAO from './dao/AvailabilityDAO';
+import BeverageDAO from './dao/BeverageDAO';
+import DeviceDAO from './dao/DeviceDAO';
+import GlassDAO from './dao/GlassDAO';
+import KegDAO from './dao/KegDAO';
+import LocationDAO from './dao/LocationDAO';
+import PermissionDAO from './dao/PermissionDAO';
+import PourDAO from './dao/PourDAO';
+import ScheduleDAO from './dao/ScheduleDAO';
+import SrmDAO from './dao/SrmDAO';
+import StyleDAO from './dao/StyleDAO';
+import TapDAO from './dao/TapDAO';
+
+type OConfig = {
+  endpoint: string,
+  headers?: Headers,
+};
+
+class DAOApi {
+  AccountDAO: typeof AccountDAO;
+  AvailabilityDAO: typeof AvailabilityDAO;
+  BeverageDAO: typeof BeverageDAO;
+  DeviceDAO: typeof DeviceDAO;
+  GlassDAO: typeof GlassDAO;
+  KegDAO: typeof KegDAO;
+  LocationDAO: typeof LocationDAO;
+  PermissionDAO: typeof PermissionDAO;
+  PourDAO: typeof PourDAO;
+  ScheduleDAO: typeof ScheduleDAO;
+  SrmDAO: typeof SrmDAO;
+  StyleDAO: typeof StyleDAO;
+  TapDAO: typeof TapDAO;
+
+  constructor({ endpoint, headers }: OConfig) {
+    oHandler().config({
+      endpoint,
+      headers: [
+        { name: 'Prefer', value: 'return=representation' },
```

Comment:
pass this header as default, since I guess we will use it everywhere, but I can  pass in brewskey.admin  in constructor instead if you want to